DigitalOffice/Services/UserService/Mappers/EditUserRequestToDbMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.RestRequests;
+
+namespace LT.DigitalOffice.UserService.Mappers
+{
+    public class EditUserRequestToDbUserMapper : IMapper<EditUserRequest, DbUser>
```

Comment:
Why such a name?

Reply:
Fixed, I've united two mappers in one UserMapper